docs/2.8.0/_toc.yml

```diff
@@ -1233,6 +1233,8 @@ subtrees:
       title: "Example Monitoring Setup"
     - file: canton/usermanual/monitoring_glossary
       title: "Glossary"
+  - file: query/pqs-user-guide
+    title: "Querying and Accessing Ledger Data"
```

Comment:
Maybe shorten to just "Querying Ledger Data"? It's long and I'm not sure "accessing" says much.